hack/check-files-alignment.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source hack/project_info.sh
+roles_yaml_path=config/rbac/role.yaml
+origin_crd_yaml_path=config/crd/bases/csi.ibm.com_ibmblockcsis.yaml
+
+check_generation (){
+  echo "check generation"
+  project_dirname=ibm-block-csi-operator
+  cd ..
+  cp -r $project_dirname ./$project_dirname-expected
+  cd $project_dirname-expected/
+  make update
+  cd ..
+  diff -qr --exclude=bin $project_dirname $project_dirname-expected/
+  rm -rf $project_dirname-expected/
+  cd $project_dirname
```

Comment:
what the last cd is for?

Reply:
to get back to the main operator directory